drivers/serial/uart_nrfx_uarte.c

```diff
@@ -189,6 +188,52 @@ static inline NRF_UARTE_Type *get_uarte_instance(const struct device *dev)
 	return config->uarte_regs;
 }
 
+#ifndef CONFIG_PINCTRL
+static void uarte_nrfx_pins_configure(const struct device *dev, bool sleep)
+{
+	const struct uarte_nrfx_config *cfg = get_dev_config(dev);
+
+	if (!sleep) {
+		if (cfg->tx_pin != NRF_UARTE_PSEL_DISCONNECTED) {
+			nrf_gpio_pin_write(cfg->tx_pin, 1);
+			nrf_gpio_cfg_output(cfg->tx_pin);
+		}
+
+		if (cfg->rx_pin != NRF_UARTE_PSEL_DISCONNECTED) {
+			nrf_gpio_cfg_input(cfg->rx_pin, cfg->rx_pull_up);
```

Comment:
`nrf_gpio_cfg_input()` needs a `nrf_gpio_pin_pull_t` as the second parameter and `cfg->rx_pull_up` is a `bool`. Same story with `cts_pull_up` below (and in `uart_nrfx_uart.c`).

Reply:
fixed